test/start_test.exs

```diff
@@ -38,10 +38,14 @@ defmodule StartTest do
                      backoff_type: :stop]
 
     Process.flag :trap_exit, true
-    assert {:error, {%Mariaex.Error{message: "failed to upgraded socket: {:tls_alert, 'unknown ca'}"}, _}} =
```

Comment:
The original tests didn't work properly for me. I suspect this was a result of the original SSL tests always being skipped, possibly having broken in the past. If I'm mistaken, this may come down to a MySQL error message inconsistency between releases.